Extensions/Effects/pixi-filters/filter-color-gradient.js

```diff
@@ -0,0 +1,150 @@
+/*!
+ * @pixi/filter-color-gradient - v5.2.0
```

Comment:
this one is 5.2.0 where the others are 5.1.1, I suppose this has no big impact?

Reply:
This is because every filter has it's own package and I guess filters versions are only incremented when they have an actual change.